common/src/main/java/org/wso2/testgrid/common/ShellExecutor.java

```diff
@@ -38,9 +42,13 @@
 
     private static Logger logger = LoggerFactory.getLogger(ShellExecutor.class);
 
-    private File workingDirectory;
+    private Path workingDirectory;
 
-    public ShellExecutor(File workingDirectory) {
+    public ShellExecutor() {
```

Comment:
What will be the usages of this constructor?

Reply:
if you do not need set a working directory you can use the default constructor.